src/lib.rs

```diff
@@ -52,6 +52,8 @@ const DLL: &[u8] =
 #[cfg(all(feature = "embed-dll", target_arch = "aarch64"))]
 const DLL: &[u8] =
     include_bytes!("..\\Microsoft.Web.WebView2.0.9.430\\build\\arm64\\WebView2Loader.dll");
+#[cfg(not(feature = "embed-dll"))]
+const DLL: &[u8] = b"";
```

Comment:
If you use `#[cfg(...)]` instead of `if cfg!(...)` in the `build` method, this shouldn't be necessary.

Reply:
thank you for all your comments